improved readme (#19)
yaptide/deploy: README.md

ansible-playbook cloud_instance.yml -e "instance_name='yaptide2'"
```

5. Deploy the yaptide platform on the instance.
At this stage necessary packages will be installed and respective repositories cloned.
The backend will be served on port 5000 and the frontend on port 80.
This step is time consuming and may take more than 10 minutes.

> We assume that you have locally a binary of `shieldhit` simulator stored in the same location as the `site.yml` playbook.

Reviewer: now that it breaks when shieldhit is not up to date we could mention it in a readme 
```suggestion
We assume you have the latest binary of the `shieldhit` simulator locally stored in the same location as the `site.yml` playbook.
```
